docs/README.md

```
@@ -265,7 +265,18 @@ systemctl restart netbox
 
 ## Uninstall
 
-See [the instructions for uninstalling plugins](https://netboxlabs.com/docs/netbox/en/stable/plugins/removal/).
+To cleanly uninstall the plugin and reverse its migrations, **while the plugin is still installed and enabled**, do the following:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)